shared/locales/de.json

```diff
@@ -0,0 +1,672 @@
+        "posts": "Keine Posts",
+        "feed": "Keine Posts",
+        "conversations": "Keine Konversationen",
+        "messages": "Keine Nachrichten",
+        "followers": "Keine Follower",
+        "following": "Niemand folgt",
+        "recommendations": "Keine Vorschläge",
+        "playlists": "Keine Playlisten",
+        "communities": "Keine Communities",
+        "members": "Keine Mitglieder",
+        "videos": "Keine Videos",
+        "related": "Keine Verwandten",
+        "lyrics": "Keine Lyrics",
+        "listeners": "Keine Zuhörer"
+      },
+      "content": "Hier scheint nichts zu sein."
+    },
+    "deleted": {
+      "artist": "Künstler wurde gelöscht",
+      "album": "Album wurde gelöscht",
+      "track": "Titel wurde gelöscht",
+      "recommendation": "Empfehlung wurde gelöscht",
+      "playlist": "Playlist wurde gelöscht",
+      "post": "Post wurde gelöscht"
+    },
+    "messages": {
+      "welcome": "Willkommen, {profileNickname}"
+    },
+    "profile": {
+      "wasOnline": "War online",
+      "since": "Nutzt muffon seit"
+    },
+    "feed": {
+      "global": "Globaler feed"
+    },
+    "about": {
+      "license": "{license} Lizenz",
+      "homepage": "Startseite",
+      "contact": "Kontakt"
+    },
+    "listeners": "{count} zuhörer | {count} zuhörer",
+    "private": {
+      "playlist": "Privat",
+      "profile": "Privat"
+    },
+    "donate": {
+      "header": "Donate",
+      "content": {
+        "nonProfit": "muffon ist ein gemeinnütziges Projekt, das weder Werbung noch kostenpflichtige Funktionen oder Inhalte hat.",
+        "funds": "Aber natürlich kann es ohne Gelder, die für seine Entwicklung und Infrastruktur benötigt werden, nicht existieren.",
+        "timeEffort": "Es wurde viel Zeit und Mühe in die Erstellung dieses Projekts gesteckt, und es wäre gut, wenn sich alle Investitionen auszahlen würden.",
+        "help": "Deshalb bitte ich um Ihre Hilfe bei der Unterstützung von muffon.",
+        "would": "Möchten sie Spenden?"
+      },
+      "decline": "Nein, danke",
+      "accept": "Ja, bitte"
+    },
+    "lists": {
+      "simple": "Simple Liste",
+      "table": "Tabelle",
+      "extended": "Detailierte Liste"
+    },
+    "orders": {
+      "createdDesc": "Neueste",
+      "createdAsc": "Älteste",
+      "updatedDesc": "Am aktuellsten",
+      "updatedAsc": "Am ältesten",
+      "artistsCountDesc": "Meiste Künstler",
+      "artistsCountAsc": "Wengiste Künstler",
+      "albumsCountDesc": "Meisten Alben",
+      "albumsCountAsc": "Wengiste Alben",
+      "tracksCountDesc": "Meiste Titel",
+      "tracksCountAsc": "Wengiste Titel",
+      "followersCountDesc": "Meiste Follower",
+      "followersCountAsc": "Wengiste Follower",
+      "joinedDesc": "Am neuesten",
+      "joinedAsc": "Am ältesten",
+      "membersCountDesc": "Meiste Mitglieder",
+      "membersCountAsc": "Wengiste Mitglieder"
+    },
+    "error": "Fehler",
+    "more": "Mehr",
+    "loading": "Lädt..."
+  }
+  
```

Comment:
@gamersi Seems like there's a trailing whitespace here.

Reply:
if you mean the one in line 672 i removed it now

Reply:
@gamersi Thanks.